openshift/cd-applications/nexus3-app.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: nexus3
+  namespace: openshift-gitops
+spec:
+  destination:
+    server: 'https://api.lab02.gpslab.club:6443'
+    namespace: demo-app
+  source:
+    path: openshift/cd-components/nexus3
+    repoURL: 'https://github.com/thekoma/k8s-demo-app.git'
```

Comment:
Refer to the parent repo if possible: we need to make sure that code is pulled from the same repository.